### PR TITLE
rework documentation for new JDK-specific goals

### PR DESCRIPTION
**Pull request by hboutemy**
publishing result to https://maven.apache.org/plugins-archives/maven-toolchains-plugin-LATEST/index.html

`mvn -Preporting site-deploy`

### CODE REVIEW

File: src/main/java/org/apache/maven/plugins/toolchain/ToolchainMojo.java [hunk 1]
```diff
@@ -62,9 +62,9 @@ public class ToolchainMojo extends AbstractMojo {
     /**
      * Toolchains requirements, specified by one
      * <pre>  &lt;toolchain-type&gt;
-     *    &lt;param&gt;expected value&lt;/param&gt;
-     *    ...
-     *  &lt;/toolchain-type&gt;</pre>
+     *     &lt;param&gt;expected value&lt;/param&gt;
+     *     ...
+     *   &lt;/toolchain-type&gt;</pre>
```
Comment (olamy) [inline src/main/java/org/apache/maven/plugins/toolchain/ToolchainMojo.java:67]: what about something such 
```
     * <pre>{@code  <toolchain-type>
     *     <param>expected value</param>
     *     ...
     *   </toolchain-type>
     * }</pre>
     *
```
I find more human readable compared to the escaped values. Should work for the goal page generation.

**Review by olamy — APPROVED** [review]
lgtm. 
just a minor comment on javadoc format.

File: src/main/java/org/apache/maven/plugins/toolchain/ToolchainMojo.java (hunk 1 above)
Comment (hboutemy) [reply src/main/java/org/apache/maven/plugins/toolchain/ToolchainMojo.java:67]: if it works, I take: I'll try and report

Comment (hboutemy) [reply src/main/java/org/apache/maven/plugins/toolchain/ToolchainMojo.java:67]: it works, good idea integrated, thank you :)

**Pull request merged**